docker-compose.yml

```diff
@@ -11,3 +11,7 @@ services:
       - "8000:8000"
     depends_on:
       - db
+    entrypoint:
```

Comment:
@johnbradley This was a ugly-but-not-too-ugly way to use the wait-for-postgres script without making it the default entrypoint. I added the script to the source repo (and therefore the docker image), but it's only activated under docker-compose. So the docker image by itself isn't bound to use it.

The original docker image does have one dependency on the compose configuration - it expects the database to be on a host called `db`. I see you extracted that out in your app, and I suppose I should too.